docs/source/hardware/remote_controllers.rst

```diff
@@ -1,3 +1,104 @@
+The control stick control the throttle, roll, pitch, and yaw of the quadrotor. The *zero configuration* is illustrated in the image below where all sticks are at the center of their range. 
+
+**Flight Mode Toggle**
+The flight mode toggle switches between stability mode (0), position hold mode (1), and offboard mode (2). 
+
+**Arm/Disarm Toggle**
+The arm/disarm toggle arms (1) or disarms (0) the quadrotor. The optional throttle cut switch cuts the throttle if moved from the (0) position. This is a last resort option to stop the motors from spinning. *DO NOT* cut throttle in flight as the quadrotor will drop to the ground. 
+
+.. image:: spektrum_configuration_front.jpg
+  :width: 310px
+  :alt: Spektrum Configuration Front
+  :align: left
+
+.. image:: spektrum_configuration_top.jpg
+  :width: 310px
+  :alt: Spektrum Configuration Top
+  :align: right
+
+Use
+------------------
+
+Please read this entire section before attempting to fly a quadrotor -- even autonomously. Familiarity with the emergency landing procedure can save quadrotors, time, and prevent injuries. 
+
+**Preflight Check**
+
+* Ensure throttle is all the way down (not in the zero configuration)
+* Ensure all switches are set to the default 0 position
+* Launch quadrotor mocap ROS nodes (the quadrotor cannot fly without pose feedback from mocap)
+
+.. TODO:: Add a link here to the mocap documentation page
+
+**Arm Procedure**
+
+* Perform preflight check
+* Arm quadrotor's safety button located on the flight controller
+* With thottle down, slide yaw control all the way to the left and hold for 2 seconds
+* Flip the arm toggle to 1
+* The drone is now armed and ready for flight
+
+**Disarm Procedure**
+
+* Flip the arm toggle to 1
+* With thottle down, slide yaw control all the way to the right and hold for 2 seconds
+* Disarm quadrotor's safety button located on the flight controller
+* The drone is now safe to be handled
+
+.. TODO:: Maybe we can refer to another page that specifies the different flight modes so we don't clutter this page? Then each of the following sections can just link to that page. 
+
+**Stability Flight**
+
+Stability mode enables you to fly the quadrotor manually using the transmitter as you would expect on a normal drone. Here, the throttle stick position is proportional to thrust. To fly manually in stability mode:
+
+* Arm the quadrotor
+* Take off by gently increasing the throttle
+
+**Position Hold Flight**
+
+Position hold flight uses mocap pose feedback to keep the quadrotor hovering at desired waypoints. These waypoints are set using the transmitter's control sticks. Here, all of the sticks *must* be set in the zero configuration (in the middle of their range). Deviation from the zero configuration modifies the new waypoint. For example, if the quadrotor is hovering and you increase the thottle stick from the zero configuration upwards, the quadrotor will fly higher. This is *different* from stability flight. To fly in position hold mode:
```

Comment:
This might split things up a bit more than we'd like, but cautions or the like might make sense in some of these subsections, as the information you are mentioning seems like they need to be highlighted. Safety 3rd, but still...